docs/guides/instances.mdx

```diff
@@ -0,0 +1,112 @@
+   </Admonition>
+
+1. From the dashboard Instances pane, click **View all**, or go to the [Instances](https://quantum.cloud.ibm.com/instances) page from the main menu.
+1. From the Instances page, click **Create instance**.
+    * Enter a name and optionally add tags.
+    * If you have access to more than one resource group in IBM Cloud&reg;, you can choose to change the resource group this instance belongs to. See [Managing resource groups](https://cloud.ibm.com/docs/account?topic=account-rgs&interface=ui) and [Giving access to resources in resource groups](https://cloud.ibm.com/docs/account?topic=account-rgs_manage_access) for more information.
+1. Select the pricing plan this instance is associated with (for example, Pay-as-you-go). Depending on the pricing plan selected, you can specify other options, such as the cost limit or the number of minutes to allocate to this instance. Note that If this instance is not associated with a free plan, a cost will be incurred when this instance is used to run workloads.
+1. Click **Next**. A list of the QPUs the instance can access is shown.
+    - If you want the instance to have access to all the QPUs listed (and all QPUs added to this plan in the future), click **Next**.
+    - If you are not setting up an open instance, you can customize which QPUs can be accessed with this instance. Click the "Customize allocated compute resources" toggle. Select specific QPUs that this instance can access, then click **Next**.
+        <Admonition type="note">
+      If you customize the allocated compute resources, the available QPUs will never be automatically updated, regardless of any changes made to the parent plan.  However, you can manually add or remove QPUs later. If you do not customize them, you will always have access to any QPU the account has access on the plan. So if in the future there is a new QPU added to the plan, the instance automatically has access to it.
+        </Admonition>
+1. The Access group pane opens. A "collaborators" access group is automatically created for this instance. Users added to this access group can use the time allocated to this instance. You can [modify or create additional access groups](/docs/guides/access-groups) by using the [IBM Cloud console.](https://cloud.ibm.com/iam/groups) See [Setting up access groups](https://cloud.ibm.com/docs/account?topic=account-groups&interface=ui) for instructions.
+1. Click **Create instance**. You can now view the instance on the [Instances](https://quantum.cloud.ibm.com/instances) page. If you have set up multiple plans on your account, each plan type has its own tab on the Instances table.
+
+## Edit an instance and its allocation
+
+Find the instance on the [Instances](https://quantum.cloud.ibm.com/instances) page. Click the overflow menu at the end the instance's row, and select Edit details.
```

Comment:
I don't think everyone is able to edit an instance. I don't have an overflow menu on the instances that I'm not an administrator on. Once we can sync/deploy new content, we should link from here to the new tables that will be in the access-groups file, pointing to what roles can do which actions.

Reply:
Tracked in an issue